kubernetes/zenko/charts/mgob/values.yaml

```diff
@@ -29,7 +29,7 @@ configMap:
       database: "metadata"
       # params: "--authenticationDatabase admin"
     scheduler:
-      cron: "0 5 * * *"
+      cron: "0 0 31 2 *"
```

Comment:
This can't be correct. Please double check this value.

Reply:
Good catch, I didn't want it to run so had changed it to 31st feb.

Reply:
fixed.